Rakefile

```diff
@@ -36,5 +37,5 @@ task :copy_man_page_to_manpath do |_t|
   end
 end
 
-task checks: [:rubocop, :rspec]
-task default: [:rubocop, :rspec]
+task checks: %i[rubocop rspec]
+task default: %i[rubocop rspec]
```

Comment:
This shines a light on an issue – but about this code: the symbol list notation was not introduced in 1.9.3. Our .gemspec notes 1.9.3 as the required Ruby version.

We do not _test_ on 1.9.3. It is EOL'd.

We still keep `- rvm: 2.1` in the Travis CI matrix. That is also EOL'd.

So, two things I think we should do, in other PRs:

- Drop `2.1` from the CI matrix
- Bump up required version in gemspec to something newer, like `">= 2.0.0"`, so that we can later merge _this_ PR. 
- Add a `TargetRubyVersion` to the .rubocop.yml, which points at the lowest tested version (currently: `"2.2.7"`)